zk-keygen/src/main.rs

```diff
@@ -158,6 +238,81 @@ fn do_main(matches: &ArgMatches) -> Result<(), Box<dyn error::Error>> {
                 }
             }
         }
+        ("pubkey", matches) => {
+            let key_type = KeyType::get_key_type(matches);
+
+            let mut path = dirs_next::home_dir().expect("home directory");
+            let path = if matches.is_present("keypair") {
+                matches.value_of("keypair").unwrap()
+            } else {
+                path.extend([".config", "solana", key_type.default_file_name()]);
+                path.to_str().unwrap()
+            };
```

Comment:
hrm... we aren't respecting a cli-config value?

Reply:
The encryption keys are not yet supported in the cli-config, so I think it doesn't make sense to use it here.

On a follow-up PR, I was hoping to add a `derive` command that will derive encryption keys from signing keys. For this command, I think it makes sense to use cli-config as default if no path is specified.

Reply:
any plans to add that?  this assumption is a bit unintuitive from how the other tools work

Reply:
Okay, I think it makes to add it into cli-config. I will look into it on a follow-up.